### PR TITLE
fix long urls breaking moderation box

### LINKED ISSUE #1835: Long urls break comment cards in Moderator view
#### Do you want to request a **feature** or report a **bug**?

Bug

#### Intended outcome:
Long urls should wrap within the comment cards in Admin > Mod Queues.

#### Actual outcome:

long_urls

#### How to reproduce the issue:
Create a new comment with a long url. You can use this: https://longurlmaker.com/

#### Version and environment
4.6.1 on Coral Staging

### PR DESCRIPTION
**Pull request by hjanuschka**
fixes: https://github.com/coralproject/talk/issues/1835

bildschirmfoto 2018-09-03 um 08 20 18

### CODE REVIEW

**Comment by kgardnr** [comment]
Thank you for fixing this @hjanuschka!!

Tagging @clintabrown as he was the one who reported this.

We will review this asap!

File: client/coral-admin/src/routes/Moderation/components/Comment.css [hunk 1]
```diff
@@ -61,6 +61,7 @@
   line-height: 1.5;
   font-weight: 300;
   margin-bottom: 8px;
+  word-break: break-all;
```
Comment (alinhle) [inline client/coral-admin/src/routes/Moderation/components/Comment.css]: Hi @hjanuschka, thanks for this PR.

`word-break: break-all;` breaks in the middle of words even if it doesn't have to. Would `word-wrap: break-word;` also do?

**Review by alinhle — CHANGES_REQUESTED** [review]

**Comment by hjanuschka** [comment]
feedback adressed 👍

File: client/coral-admin/src/routes/Moderation/components/Comment.css [hunk 2]
```diff
@@ -61,7 +61,7 @@
   line-height: 1.5;
   font-weight: 300;
   margin-bottom: 8px;
-  word-break: break-all;
+  word-break: break-word;
```
Comment (cvle) [inline client/coral-admin/src/routes/Moderation/components/Comment.css]: Even better break is `overflow-wrap: break-word;` which only breaks word if absolute necessary.

See note in, which explains the contrast to `word-break`.
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

File: client/coral-admin/src/routes/Moderation/components/Comment.css [hunk 3]
```diff
@@ -61,6 +61,7 @@
   line-height: 1.5;
   font-weight: 300;
   margin-bottom: 8px;
+  word-break: break-word;
```
Comment (alinhle) [inline client/coral-admin/src/routes/Moderation/components/Comment.css]: Even better break is overflow-wrap: break-word; which only breaks word if absolutely necessary.

See note in, which explains the contrast to word-break.
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

**Review by alinhle — CHANGES_REQUESTED** [review]

**Comment by hjanuschka** [comment]
@cvle nice, didnt' know that one, thx, feedback addressed

**Review by alinhle — APPROVED** [review]

**Comment by alinhle** [comment]
Thanks!

**Pull request merged**